exporter/exporterparser/zipkin.go

```diff
@@ -27,20 +28,51 @@ import (
 	"github.com/census-instrumentation/opencensus-service/exporter"
 )
 
-type zipkinConfig struct {
+type ZipkinConfig struct {
 	ServiceName      string `yaml:"service_name,omitempty"`
 	Endpoint         string `yaml:"endpoint,omitempty"`
 	LocalEndpointURI string `yaml:"local_endpoint,omitempty"`
 }
 
+// zipkinExporter is a multiplexing exporter that spawns a new OpenCensus-Go Zipkin
+// exporter per unique node encountered. This is because serviceNames per node define
+// unique services, alongside their IPs. Also it is useful to intercept traffic from
+// Zipkin servers and then transform them back to the final form when creating an
+// OpenCensus spandata.
 type zipkinExporter struct {
-	exporter *zipkin.Exporter
+	mu sync.RWMutex
```

Comment:
It will be nice to have a comment of it is going to be protected by the mutex.

Reply:
Great idea, thanks.

Reply:
Done, thanks @pjanotti!